### PR TITLE
Kafka

### LINKED ISSUE #17: kafka

### PR DESCRIPTION
**Pull request by eyakauleva**
close #17

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+services:
```
Comment (h1alexbel) [inline docker-compose.yaml:1]: move to k8s

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -10,6 +10,19 @@ spring:
     default-property-inclusion: non_null
   application:
     name: USER-SERVICE
+  kafka:
+    topic: users
+    partitions: 3
+    replicas: 1
+    bootstrap-servers: localhost:9092
```
Comment (h1alexbel) [inline src/main/resources/application.yaml:17]: host is hardcoded

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml (hunk 2 above)
Comment (eyakauleva) [reply src/main/resources/application.yaml]: fix in 255168a

File: docker-compose.yaml (hunk 1 above)
Comment (eyakauleva) [reply docker-compose.yaml:1]: I will implement k8s in the next PR

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**